relay/helper/model_mapped.go

```
@@ -4,17 +4,43 @@ import (
+	return modelName
+}
+
 func ModelMappedHelper(c *gin.Context, info *common.RelayInfo, request dto.Request) error {
 	if info.ChannelMeta == nil {
 		info.ChannelMeta = &common.ChannelMeta{}
 	}
 
+	// Strip per-channel model prefix from the upstream model name.
+	// The frontend prepends the channel's model_prefix when saving models,
+	// but the upstream API expects the model name without the prefix.
+	if info.ChannelMeta.ChannelOtherSettings.ModelPrefix != "" {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "ChannelMeta" from selector

(staticcheck)